app/views/devise/registrations/edit.html.erb

```diff
@@ -1,43 +1,32 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
-
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
-
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+<div class="profile_show_and_edit">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+    <%= devise_error_messages! %>
+
+    <div class="tabbed">
+      <div class="label">user:</div>
+      <%= @user.name %>
+    </div>
+    <div class="tabbed">
+      <div class="label">created:</div>
+      <%= @user.created_at %>
```

Comment:
I recommend using `resource` or `f.object` here instead of `@user` to make sure this is using the same resource that the form is using.